src/index.html

```diff
@@ -69,5 +69,24 @@
         <svg class="loading" viewBox="25 25 50 50"></svg>
       </div>
     </app-root>
+
+    <!-- Adicionando botão simples em todas as páginas para acesso ao formulário de avaliação do QD -->
+    <section id="c-toast-section" class="c-toast">
+      <div class="c-toast__actions">
+        <button class="c-toast__close" onclick="closeButton()">
+          <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M11.0327 8L15.5814 3.45136C16.1395 2.89318 16.1395 1.98818 15.5814 1.42955L14.5705 0.418636C14.0123 -0.139545 13.1073 -0.139545 12.5486 0.418636L8 4.96727L3.45136 0.418636C2.89318 -0.139545 1.98818 -0.139545 1.42955 0.418636L0.418636 1.42955C-0.139545 1.98773 -0.139545 2.89273 0.418636 3.45136L4.96727 8L0.418636 12.5486C-0.139545 13.1068 -0.139545 14.0118 0.418636 14.5705L1.42955 15.5814C1.98773 16.1395 2.89318 16.1395 3.45136 15.5814L8 11.0327L12.5486 15.5814C13.1068 16.1395 14.0123 16.1395 14.5705 15.5814L15.5814 14.5705C16.1395 14.0123 16.1395 13.1073 15.5814 12.5486L11.0327 8Z"/>
+          </svg>
+        </button>
+      </div>
+      <div>
+        <h6 class="c-toast__title">Ajude a melhorar ainda mais sua experiência com a nossa plataforma!</h6>
+        <p class="c-toast__desc">Responda o formulário a seguir, leva só 5 minutos, e vai ser muito importante para gente planejar melhorias úteis para todas(os)</p>
+        <a href="http://go.ok.org.br/pesquisa-qd" title="Responda nossa pesquisa de satisfação" class="btn-secondary c-toast__btn">
```

Comment:
Sugiro colocar esse atributo para o formulário abrir em uma nova aba.

```suggestion
        <a href="http://go.ok.org.br/pesquisa-qd" target="_blank" title="Responda nossa pesquisa de satisfação" class="btn-secondary c-toast__btn">
```